### PR TITLE
deployment/argocd updates

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- This is a chore
- It updates the image tags for the Helm values

### CODE REVIEW

**Comment by Baalmart** [comment]
Hi @123MwanjeMike , 

1. PLEASE add a work in progress to this PR.
2. Also, please address the existing merge conflicts.

**Comment by 123MwanjeMike** [comment]
# **PLEASE DO NOT DELETE THIS BRANCH!!!**

**Comment by 123MwanjeMike** [comment]
# *REOPENING PR TO DISABLE BRANCH DELETION*

**Review by NoahNsimbe — APPROVED** [review]
LGTM

File: src/device-registry/bin/www [hunk 1]
```diff
@@ -3,258 +3,270 @@
...
-    await consumer.subscribe({ topic: constants.HOURLY_MEASUREMENTS_TOPIC, fromBeginning: true });
-    await consumer.run({
-        eachMessage: async({ topic, partition, message }) => {
-            if (isEmpty(message)) {
-                logger.error(
-                    `KAFKA: the sent message in undefined --- ${JSON.stringify(
-                        message
+const run = async () => {
+  await consumer.connect();
+  await consumer.subscribe({
```
Comment (Baalmart) [inline src/device-registry/bin/www:98]: I do not understand why the PR is making styling changes to files/microservices that are not within the scope of the PR? @123MwanjeMike

**Review by Baalmart — CHANGES_REQUESTED** [review]
Hi @123MwanjeMike , I do notice that this PR makes some styling changes to three micros:

1. airflow
2. device-monitoring
3. device-registry

Could we avoid doing such? Just to prevent unintended "bugs"?

File: src/device-registry/bin/www (hunk 1 above)
Comment (123MwanjeMike) [reply src/device-registry/bin/www:98]: Hi @Baalmart 
Those changes were made by the linter in the pre-commit hook.

Comment (123MwanjeMike) [reply src/device-registry/bin/www]: I've removed all the linter changes @Baalmart

**Review by Baalmart — APPROVED** [review]
thanks @123MwanjeMike

**Pull request merged**